Update draft-orders.md (#11857)
medusajs/medusa: www/apps/docs/content/modules/orders/draft-orders.md

Since the draft order is associated with a cart, the process implemented in the Medusa backend around completing the draft order is pretty similar to that of completing a cart.

> First initilize and set the payment session using `CartService`'s [authorizePayment method](../../references/services/classes/services.CartService.mdx#setpaymentsessions) and [authorizePayment method](../../references/services/classes/services.CartService.mdx#setpaymentsession) methods respectivly.

Reviewer: ```suggestion
First initilize and set the payment session using `CartService`'s [setPaymentSessions method](../../references/services/classes/services.CartService.mdx#setpaymentsessions) and [authorizePayment method](../../references/services/classes/services.CartService.mdx#setpaymentsession) methods respectivly.
```

> The payment must then be authorized before the cart can be completed, which can be done using the `CartService`'s [authorizePayment method](../../references/services/classes/services.CartService.mdx#authorizepayment). Once the payment is authorized, the order can be created using the `OrderService`'s [createFromCart method](../../references/services/classes/services.OrderService.mdx#createfromcart).

Reviewer: ```suggestion
Once the payment is authorized, the cart can be completed and the order can be created using the `OrderService`'s [createFromCart method](../../references/services/classes/services.OrderService.mdx#createfromcart).
```

Author: Thanks for that!
